src/option_model.py

```diff
@@ -18,6 +18,10 @@ def create_option_model(name: str) -> _OptionModelBase:
         return _OracleOptionModel(env)
     if name == "behavior_oracle":
         return _BehaviorOptionModel()  # pragma: no cover
+    if name.startswith("oracle"):
+        _, env_name = name.split("_")
```

Comment:
Let's change this to `env_name = name[name.index("_")+1:]` in anticipation of future things like oracle_cover_regrasp